README.rst

```diff
@@ -1,7 +1,26 @@
+flit
```

Comment:
Sorry. I like the added colour of the logo, and if it could be floated to the right so it wasn't pushing the text further down, I'd be happy to have it in the readme. But as it can't, my preference is to leave it out and put up with the readme looking a bit dull.